.github/workflows/ci.yml

```diff
@@ -29,14 +29,9 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-22.04]
-        scala: [2.13, 3]
-        java: [temurin@8, temurin@11, temurin@17]
+        scala: [2.12, 2.13, 3]
+        java: [temurin@17]
```

Comment:
Please check out [this comment](https://github.com/http4s/http4s-servlet/pull/399/files#r2170779564).